__tests__/api.test.js

```diff
@@ -0,0 +1,22 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+const topics = require('../db/data/test-data/topics')
+
+beforeAll(() => seed(data))
```

Comment:
This does work for now but will pose a problem when you get to a post request. This is because the seed will only run once before all the tests are run. `beforeEach` will serve you better!